packages/webapp/public/images/template-logos/strava-web.svg

```diff
@@ -0,0 +1 @@
+strava.svg
```

Comment:
Thank you for your contribution. the svg looks malformed. Can you fix it?

Reply:
![image](https://github.com/NangoHQ/nango/assets/39801641/4b32fbc3-33b1-4845-8e09-580958632ad7)
The new image is created as a sym link. 

Similar to stripe-app-sandbox.svg
![image](https://github.com/NangoHQ/nango/assets/39801641/4f71f14a-8ca5-423a-a837-38435628d976)

Please suggest the required change.

Reply:
My bad. I read it as it was the content of the svg not a symlink.

Reply:
No problem and thank you for your swift approval.